mtgjson5/providers/tcgplayer.py

```diff
@@ -17,6 +18,28 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class CardFinish(enum.Enum):
+    """
+    TCGPlayer Card Condition
+    Self Driven by MTGJSON
+    """
+
+    ALTERNATE_ART = "Alternate Art"
+    BORDERLESS = "Borderless"
+    EXTENDED_ART = "Extended Art"
+    FOIL_ETCHED = "Foil Etched"
+    GLOSSY = "SDCC 2019 Exclusive"
+    RETRO = "Retro Frame"
```

Comment:
i don't get why all these variants are needed, especially because they are not technically "finishes", but different frame effects/borders/frame version

I think you could just check if the name contains "foil etched" and mark it as such, no?

Reply:
Is Foil Etched the only entity we want to support? I'd argue Glossy should be supported too. I understand why you might be against retro/extended, so i'll ask a few others about it

Reply:
yes, foil etched is the only one entity we need. Glossy is tame so far as there are no duplicates finishes, so there is no need to duplicate them (and I'd argue we shouldn't unless they became a real thing from wotc)

note that I am not "against retro/extended", they simply are not finishes, each of those will have separate uuids, so there is no need to exposing them in this file like this